x-pack/plugins/apm/server/routes/services.ts

```diff
@@ -397,12 +405,15 @@ export const serviceInstancesRoute = createRoute({
     const setup = await setupRequest(context, request);
     const { serviceName } = context.params.path;
     const { transactionType, numBuckets } = context.params.query;
+    const latencyAggregationType = (context.params.query
+      .latencyAggregationType as unknown) as LatencyAggregationType;
```

Comment:
Why do you need `as unknown` here?

Reply:
Not sure. I get this:

```
const latencyAggregationType: LatencyAggregationType
Type '"avg" | "p95" | "p99"' is not assignable to type 'LatencyAggregationType'.
  Type '"avg"' is not assignable to type 'LatencyAggregationType'.ts(2322)
```

Reply:
I think `as LatencyAggregationType ` is enough, isn't it?

Reply:
No that's what I get when I use `as LatencyAggregationType`.